docs/Configuring-Pantheon/Consensus-Mechanisms.md

```diff
@@ -46,9 +44,19 @@ increases number as the of validators increases.
 
 For IBFT 2.0, the time to add new blocks increases as the number of validators increases.   
 
-### Minimum Number of Validators 
+### Type of Byzantine-fault-tolerance
+
+IBFT 2.0 features a classical BFT consensus protocol that ensures safety (no fork is possible) provided that no more than
+(n-1)/3 of the validators (truncated to the integer value) are Byzantine.
+For example in an IBFT 2.0 network of:
+* 3, no Byzantine nodes are tolerated
+* 4-6, 1 Byzantine node is tolerated
```

Comment:
I have heard murmerings of an edgecase where ibft 2.0 does not perform well.  Is that true or not?

Reply:
@saltiniroberto - see question above from Rob

Reply:
I discussed this with Rob and it is all good. There are no known edge cases with IBFT 2.0